pkgs/cli/src/main.zig

```diff
@@ -216,8 +216,8 @@ pub fn main() !void {
             var validator_ids_1 = [_]usize{1};
             var validator_ids_2 = [_]usize{2};
 
-            const logger1 = utilsLib.getScopedLogger(.n1, .debug);
-            const logger2 = utilsLib.getScopedLogger(.n2, .debug);
+            const logger1 = utilsLib.getScopedLogger(.n1, .debug, "./log");
```

Comment:
are the two loggers able to interleave into the same file?

also add the file name as a parameter in the cli args but with a default, and default file name should be `consensus.log`